src/main/java/org/ccci/obiee/client/rowmap/impl/AnalyticsManagerImpl.java

```diff
@@ -169,9 +175,17 @@ public void close()
     {
         checkOpen();
         closed = true;
-        log.debug("logging off session " + sessionId);
-        sawSessionService.logoff(sessionId);
-        log.debug("logoff successful");
+        final Span span = buildTopLevelSpan(tracer, "logoff");
+        try (Scope ignored = tracer.scopeManager().activate(span, false))
```

Comment:
Yes.

You can't use a try-with-resources block without assigning to a variable.
At the same time, this code doesn't use the resulting scope. And Intellij gives me a warning unless I name it 'ignored'.

Reply:
Since I wrote this, I've read that some people consider this abusing the try-with-resources feature, and the next version of open tracing will actually deprecate this.

Reply:
Actually, what I just said isn't true. They're deprecating `activate(span, true)`, which finishes the span when the scope is closed, and which this project doesn't use. 

They're not deprecating try-with-resources for scopes. And I think this is a decent use for scopes, so I'm glad they're not.